src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -54,14 +57,67 @@ void createStation() {
      * Given 2개의 지하철역을 생성하고
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
+     * 인수 테스트란, End to End Test 인데, Controller -> Service -> Repository 까지 테스트 흐름이 움직이기 때문에 통합 테스트와 비슷한 성격을 띄는 것 같은데, 맞나...?
      */
     // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역 목록을 조회한다.")
+    @Test
+    void findAllStations() {
+        // Given
+        saveStation("지하철역이름");
+        saveStation("새로운지하철역이름");
+        saveStation("또다른지하철역이름");
+
+        // 조회
+        ExtractableResponse<Response> response = RestAssured
+                .given()
+                .when().get("/stations")
+                .then()
+                .extract();
+
+        List<StationResponse> stationResponses = response.jsonPath().getList(".", StationResponse.class);
+        assertThat(stationResponses.size()).isEqualTo(3);
+    }
+
+    private static void saveStation(String name) {
+        Map<String, String> param = new HashMap<>();
+        param.put("name", name);
+
+        RestAssured.given()
+                .body(param)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().post("/stations")
+                .then()
+                .statusCode(HttpStatus.CREATED.value());
+    }
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
     // TODO: 지하철역 제거 인수 테스트 메서드 생성
+    @DisplayName("지하철역을 제거한다.")
+    @Test
+    void deleteStation() {
+        // Given
+        saveStation("지하철역이름");
+        saveStation("새로운지하철역이름");
+        saveStation("또다른지하철역이름");
+
+        // When
+        RestAssured
+                .given().log().all()
+                .when().delete("/stations/1")
+                .then().log().all();
 
-}
\ No newline at end of file
+        // then
+        List<StationResponse> stationResponses = RestAssured
+                .given()
```

Comment:
로그를 찍으면 테스트의 상황을 조금 더 잘 파악할 수 있습니다 :) 

```suggestion
                .given().log().all()
```